affected_purls:
- pkg:pypi/pymongo@0.10
- pkg:pypi/pymongo@0.10.1
- pkg:pypi/pymongo@0.10.2
- pkg:pypi/pymongo@0.10.3
- pkg:pypi/pymongo@0.11
- pkg:pypi/pymongo@0.11.1
- pkg:pypi/pymongo@0.11.2
- pkg:pypi/pymongo@0.11.3
- pkg:pypi/pymongo@0.1.1pre
- pkg:pypi/pymongo@0.12
- pkg:pypi/pymongo@0.1.2pre
- pkg:pypi/pymongo@0.13
- pkg:pypi/pymongo@0.14
- pkg:pypi/pymongo@0.14.1
- pkg:pypi/pymongo@0.14.2
- pkg:pypi/pymongo@0.15
- pkg:pypi/pymongo@0.15.1
- pkg:pypi/pymongo@0.15.2
- pkg:pypi/pymongo@0.16
- pkg:pypi/pymongo@0.1pre
- pkg:pypi/pymongo@0.2pre
- pkg:pypi/pymongo@0.3.1pre
- pkg:pypi/pymongo@0.3pre
- pkg:pypi/pymongo@0.4pre
- pkg:pypi/pymongo@0.5.1pre
- pkg:pypi/pymongo@0.5.2pre
- pkg:pypi/pymongo@0.5.3pre
- pkg:pypi/pymongo@0.5pre
- pkg:pypi/pymongo@0.6
- pkg:pypi/pymongo@0.7
- pkg:pypi/pymongo@0.7.1
- pkg:pypi/pymongo@0.7.2
- pkg:pypi/pymongo@0.8
- pkg:pypi/pymongo@0.8.1
- pkg:pypi/pymongo@0.9
- pkg:pypi/pymongo@0.9.1
- pkg:pypi/pymongo@0.9.2
- pkg:pypi/pymongo@0.9.3
- pkg:pypi/pymongo@0.9.4
- pkg:pypi/pymongo@0.9.5
- pkg:pypi/pymongo@0.9.6
- pkg:pypi/pymongo@0.9.7
- pkg:pypi/pymongo@1.0
- pkg:pypi/pymongo@1.1
- pkg:pypi/pymongo@1.10
- pkg:pypi/pymongo@1.10.1
- pkg:pypi/pymongo@1.1.1
- pkg:pypi/pymongo@1.11
- pkg:pypi/pymongo@1.1.2
- pkg:pypi/pymongo@1.2
- pkg:pypi/pymongo@1.2.1
- pkg:pypi/pymongo@1.3
- pkg:pypi/pymongo@1.4
- pkg:pypi/pymongo@1.5
- pkg:pypi/pymongo@1.5.1
- pkg:pypi/pymongo@1.5.2
- pkg:pypi/pymongo@1.6
- pkg:pypi/pymongo@1.7
- pkg:pypi/pymongo@1.8
- pkg:pypi/pymongo@1.8.1
- pkg:pypi/pymongo@1.9
- pkg:pypi/pymongo@2.0
- pkg:pypi/pymongo@2.0.1
- pkg:pypi/pymongo@2.1
- pkg:pypi/pymongo@2.1.1
- pkg:pypi/pymongo@2.2
- pkg:pypi/pymongo@2.2.1
- pkg:pypi/pymongo@2.3
- pkg:pypi/pymongo@2.4
- pkg:pypi/pymongo@2.4.1
- pkg:pypi/pymongo@2.4.2
- pkg:pypi/pymongo@2.5
- pkg:pypi/pymongo@2.5.1
aliases:
- CVE-2013-2132
- GHSA-x33v-f3gp-gw2c
fixed_purl:
- pkg:deb/debian/pymongo@2.5.2-1?distro=sid
- pkg:deb/debian/pymongo@3.11.0-1?distro=sid
- pkg:deb/debian/pymongo@3.7.1-1.1?distro=sid
- pkg:pypi/pymongo@2.5.2
references:
- id: 135274
  reference_id: ''
  url: https://github.com/mongodb/mongo-python-driver/commit/a060c15ef87e0f0e72974c7c0e57fe811bbd06a2
- id: 135277
  reference_id: ''
  url: https://github.com/pypa/advisory-database/tree/main/vulns/pymongo/PYSEC-2013-30.yaml
- id: 135275
  reference_id: ''
  url: https://jira.mongodb.org/browse/PYTHON-532
- id: 135278
  reference_id: ''
  url: https://lists.opensuse.org/opensuse-updates/2013-06/msg00180.html
- id: 135279
  reference_id: ''
  url: https://seclists.org/oss-sec/2013/q2/447
- id: 135280
  reference_id: ''
  url: https://ubuntu.com/usn/usn-1897-1
- id: 135281
  reference_id: ''
  url: https://www.debian.org/security/2013/dsa-2705
- id: 135282
  reference_id: ''
  url: https://www.osvdb.org/93804
- id: 135283
  reference_id: ''
  url: https://www.securityfocus.com/bid/60252
- id: 125040
  reference_id: '710597'
  url: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=710597
- id: 135273
  reference_id: CVE-2013-2132
  url: https://nvd.nist.gov/vuln/detail/CVE-2013-2132
- id: 135276
  reference_id: GHSA-x33v-f3gp-gw2c
  url: https://github.com/advisories/GHSA-x33v-f3gp-gw2c
severities:
- id: 18595
  reference_id: 135276
  scoring_elements: ''
  scoring_system: cvssv3.1_qr
  value: MODERATE
summary: bson/_cbsonmodule.c in the mongo-python-driver (aka. pymongo) before 2.5.2,
  as used in MongoDB, allows context-dependent attackers to cause a denial of service
  (NULL pointer dereference and crash) via vectors related to decoding of an "invalid
  DBRef."
vulnerability_id: VCID-4bkh-nzkg-aaaj
weaknesses: []
